packages/fxa-auth-server/lib/senders/emails/layouts/subscription/index.stories.ts

```diff
@@ -17,6 +17,13 @@ const createStory = subplatStoryWithProps(
   }
 );
 
+export const LayoutNoProduct = createStory(
+  {
+    reminderShortForm: true,
+  },
+  'Reminder short form'
```

Comment:
What does this mean, short form? Like abbreviated layout? I don't know the answer, not sure if we need more context here or not, but we could specify like, "Reminder short form, shows/hides X links" to make it more obvious what this is.